examples/ping6d/README.md

```diff
@@ -0,0 +1,15 @@
+# Ping6 daemon example
```

Comment:
obviously, this now needs a minor update.

Reply:
rebased and updated.